src/OpenSearch.OpenSearch.Ephemeral/Tasks/InstallationTasks/InitialConfigurationOpenSearch.cs

```diff
@@ -35,7 +37,10 @@ internal class InitialConfigurationOpenSearch
 		// the server what we want to do on our own, it is decided to have a
 		// snapshot of this file.
 		// The script is taken from v.1.2.4, last 3 lines omitted.
-		public static string GetConfigurationScript() =>
+		public static string GetConfigurationScript(OpenSearchVersion version)
+		{
+			if (version < (OpenSearchVersion)"2.0.0")
```

Comment:
Just not familiar with the operator. Will this actually evaluate as expected given this is a String comparison? Will the SemVer representation be understood by "<" so that if version is 1.1.1 this will be "true"?

Reply:
`OpenSearchVersion` class inherits `SemVer.Version` which is comparable - it has all operators overloaded.
see `src/OpenSearch.Stack.ArtifactsApi/OpenSearchVersion.cs`